misc/python/materialize/mzcompose.py

```diff
@@ -1807,6 +1824,35 @@ def run(self, workflow: Workflow) -> None:
             raise errors.Failed("giving up: {}".format(ui.shell_quote(self._command)))
 
 
+@Steps.register("exec")
```

Comment:
Initially I was using such an exec step to inject SQL commands into the SQL Server container, before I coded in the testdrive syntax. I think it will be handy in the future so I decided to leave it in.